fuse_models/include/fuse_models/parameters/odometry_2d_publisher_params.h

```diff
@@ -70,7 +70,7 @@ struct Odometry2DPublisherParams : public ParameterBase
    *
    * @param[in] nh - The ROS node handle with which to load parameters
    */
-  void loadFromROS(const ros::NodeHandle& nh) final
+  void loadFromROS(const ros::NodeHandle& nh) final  // TODO(CH3): Replace with a NodePtr
```

Comment:
NodeParamsInterface probably

Reply:
Comment updated https://github.com/locusrobotics/fuse/pull/279/commits/771e4f4b228a1bb1c4dea24ad07fbcd46502ea71

Pending merge of https://github.com/ros2/rclcpp/pull/2041 , but if we expect that to take too long I can duplicate an implementation in fuse_core...

Reply:
Reverting the comment update since it turns out we do have to store a SharedPtr to a node after all (for now at least)